cmd/base-keys.go

```diff
@@ -27,34 +29,34 @@ const (
 var policyKasRegistryBaseKeysCmd *cobra.Command
 
 func getKasKeyIdentifier(c *cli.Cli) (*kasregistry.KasKeyIdentifier, error) {
-	keyID := c.Flags.GetOptionalString("keyId")
-	kasID := c.Flags.GetOptionalString("kasId")
-	kasName := c.Flags.GetOptionalString("kasName")
-	kasURI := c.Flags.GetOptionalString("kasUri")
+	// This function is called when the user provides a human-readable key ID
+	// via the --key flag and identifies the KAS it belongs to via the --kas flag.
+	humanReadableKeyID := c.Flags.GetRequiredString("key") // Flag for the key's human-readable ID
+	kasIdentifierInput := c.Flags.GetRequiredString("kas") // Flag for KAS ID, URI, or Name
+
+	// Basic validation, though GetRequiredString should handle empty inputs.
+	if humanReadableKeyID == "" {
+		return nil, errors.New("--key (human-readable key ID) cannot be empty")
+	}
+	if kasIdentifierInput == "" {
+		return nil, errors.New("--kas (KAS identifier) cannot be empty")
+	}
```

Comment:
Is `GetRequiredString` not doing its job validating these? https://github.com/opentdf/otdfctl/blob/ebe260f5716a8d31ae198b1872b986cfad6299e1/pkg/cli/flagValues.go#L28